dags/hivemind_etl_helpers/src/db/github/extract/github_comment_extraction.py

```diff
@@ -0,0 +1,141 @@
+import logging
+from datetime import datetime
+
+import neo4j
+from github.neo4j_storage.neo4j_connection import Neo4jConnection
+from hivemind_etl_helpers.src.db.github.schema import GitHubComment
+
+logging.basicConfig(
+    level=logging.DEBUG,
+    filename="app.log",
+    filemode="w",
+    format="%(name)s - %(levelname)s - %(message)s",
+)
+
+
+class GitHubCommentExtraction:
+    def __init__(self):
+        """
+        Initializes the GitHubCommentExtraction class
+        without requiring any parameters.
+        Establishes a connection to the Neo4j database.
+        """
+        self.neo4j_connection = Neo4jConnection()
+        self.neo4j_driver = self.neo4j_connection.connect_neo4j()
+
+    def __fetch_raw_comments(
```

Comment:
Was there any specific reason we used two underline prefix for this method? I think the private method was using one underline. If you agree with me, please consider updating this line.